src/docs/markdown/build.md

```diff
@@ -4,10 +4,24 @@ title: "Build from source"
 
 # Build from source
 
+There are 2 ways to build Caddy, as opposed to installation of pre-built executable:
+- Build from Git repo
+- Build using `xcaddy`
```

Comment:
Let's linkify these?

Also maybe we should add a `(recommended)` note for `xcaddy` here? Vast majority should be using `xcaddy` instead of from source.

Reply:
How about:

```
- [Git](#git): Build from Git repo
- [`xcaddy`](#xcaddy): Build using `xcaddy` (Recoomended)
```

? I'll change the section headers to `Git` and `xcaddy`

Reply:
Sure 👍